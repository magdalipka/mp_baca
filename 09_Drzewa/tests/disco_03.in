1
28
CREATE PREORDER 10 44 Anna Kowalska 43 Bartek Bartek 35 Celina Sura 29 Daniel Golba 12 Ewa Kot 
31 Malgorzata Kowalczyk 38 Grzeorz Rokosz 48 Iwona Iwona 47 Jola Jolajna 50 Koko Jambo 51 Uga Buga
POSTORDER
PREORDER
INORDER
DEQUEMAX
DEQUEMIN
INORDER
NEXT 29
NEXT 47
PREV 51
DELETE 50
HEIGHT
DELETE 35
PREORDER
INORDER
NEXT 38
ENQUE 10 Halina Kiepska
PREV 10
DELETE 38
DELETE 31
INORDER
NEXT 29
ENQUE 38 Nowy Ktos
ENQUE 39 Jeszcze Nowszy 
NEXT 39
DEQUEMAX
DEQUEMIN
HEIGHT
