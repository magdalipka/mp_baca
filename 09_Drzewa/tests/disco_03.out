ZESTAW 1
POSTORDER: 12 - Ewa Kot, 31 - Malgorzata Kowalczyk, 29 - Daniel Golba, 38 - Grzeorz Rokosz, 35 - Celina Sura, 43 - Bartek Bartek, 47 - Jola Jolajna, 50 - Koko Jambo, 48 - Iwona Iwona, 44 - Anna Kowalska
PREORDER: 44 - Anna Kowalska, 43 - Bartek Bartek, 35 - Celina Sura, 29 - Daniel Golba, 12 - Ewa Kot, 31 - Malgorzata Kowalczyk, 38 - Grzeorz Rokosz, 48 - Iwona Iwona, 47 - Jola Jolajna, 50 - Koko Jambo
INORDER: 12 - Ewa Kot, 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 35 - Celina Sura, 38 - Grzeorz Rokosz, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona, 50 - Koko Jambo
DEQUEMAX: 50 - Koko Jambo
DEQUEMIN: 12 - Ewa Kot
INORDER: 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 35 - Celina Sura, 38 - Grzeorz Rokosz, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona
NEXT 29: 31 - Malgorzata Kowalczyk
NEXT 47: 48 - Iwona Iwona
PREV 51: BRAK
DELETE 50: BRAK
HEIGHT: 4
PREORDER: 44 - Anna Kowalska, 43 - Bartek Bartek, 38 - Grzeorz Rokosz, 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 48 - Iwona Iwona, 47 - Jola Jolajna
INORDER: 29 - Daniel Golba, 31 - Malgorzata Kowalczyk, 38 - Grzeorz Rokosz, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona
NEXT 38: 43 - Bartek Bartek
PREV 10: BRAK
INORDER: 10 - Halina Kiepska, 29 - Daniel Golba, 43 - Bartek Bartek, 44 - Anna Kowalska, 47 - Jola Jolajna, 48 - Iwona Iwona
NEXT 29: 43 - Bartek Bartek
NEXT 39: 43 - Bartek Bartek
DEQUEMAX: 48 - Iwona Iwona
DEQUEMIN: 10 - Halina Kiepska
HEIGHT: 4